src/main/java/com/company/verifytests/MyInfoPage.java

```diff
@@ -0,0 +1,101 @@
+package com.company.verifytests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class MyInfoPage {
+    public static WebDriver driver;
+    @FindBy(xpath = "//input[@id='btnSave']")
+    private WebElement editSaveBtn;
+
+    @FindBy(xpath = "//input[@id='personal_txtEmpFirstName']")
+    private WebElement firstNameInput;
+
+    @FindBy(xpath = "//input[@id='personal_txtEmpLastName']")
+    private WebElement lastNameInput;
+
+    @FindBy(xpath = "//input[@id='personal_txtEmployeeId']")
+    private WebElement employeeIdInput;
+
+    @FindBy(xpath = "//input[@id='personal_txtOtherID']")
+    private WebElement otherId;
+
+    @FindBy(xpath = "//input[@id='personal_txtLicenNo']")
+    private WebElement driverNumber;
+
+    @FindBy(xpath = "//input[@id='personal_optGender_2']")
+    private WebElement genderStatus;
+
+    @FindBy(xpath = "//option[contains(text(),'Ukrainian')]")
+    private WebElement nationality;
+
+    @FindBy(xpath = "//option[contains(text(),'Other')]")
+    private WebElement maritalStatus;
+
+    @FindBy(xpath = "//input[@id='personal_DOB']")
+    private WebElement birthDate;
+
+
+    public MyInfoPage(WebDriver driver){
+        PageFactory.initElements(driver, this);
+    }
+
+    public MyInfoPage clickEditSave(){
+        editSaveBtn.click();
+        return new MyInfoPage(driver);
+    }
```

Comment:
No needs to return new MyInfoPage instance, you can return `this` context if your method does not navigate user to other page.